lib/fireblocks/createtransaction.go

```diff
@@ -15,7 +15,7 @@ type TransactionRequestOptions struct {
 // CreateTransaction creates a new transaction on the FireBlocks API.
 func (c Client) CreateTransaction(ctx context.Context, opt TransactionRequestOptions) (*TransactionResponse, error) {
```

Comment:
Follow our go style guide, we don't return pointers, change to: `(TransactionResponse, error)`